app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java

```diff
@@ -70,6 +70,15 @@ public String saveCurrentUsername(String username) {
         return null;
     }
 
+    public String saveCurrentSchoolId(String schoolId) {
+        SharedPreferences.Editor editor = mPref.edit();
+        editor.putString("currentSchool", schoolId);
+
+        if (editor.commit())
+            return schoolId;
+        return null;
```

Comment:
I see, but it doesn't mean it makes sense. :D Guess this should be refactored in future.